packages/core/src/types-hoist/samplingcontext.ts

```diff
@@ -35,9 +36,9 @@ export interface SamplingContext extends CustomSamplingContext {
   location?: WorkerLocation;
 
   /**
-   * Object representing the incoming request to a node server.
+   * Object representing the incoming request to a node server in a normalized format.
    */
-  request?: ExtractedNodeRequestData;
+  normalizedRequest?: RequestEventData;
```

Comment:
m: Let's add a note to the migration guide that `request` was renamed and that the types changed. Users should check for the definedness of `request` anyway already but it's still good to know.

Reply:
I already added it [here](https://github.com/getsentry/sentry-javascript/pull/14902/files#diff-89a5adece48492133e05c0a4459cee6d1412669b37a2f39b97fa7ad385d62af6R220) :) 

But there is one dash too many in the list. I still have to update this anyway. Do you think the section is fitting or should I add it somewhere else?